packages/bpk-component-swap-button/src/BpkSwapButton.module.scss

```diff
@@ -35,23 +27,29 @@
 
   &__button {
     display: flex;
-    width: 100%;
-    height: 100%;
+    width: bpk-spacing-xxl();
+    height: bpk-spacing-xxl();
     justify-content: center;
     align-items: center;
+    transform: rotate(90deg);
```

Comment:
previously this also had a translate Y -50%. Removed this as I think it was an overhang from this originally coming in with copied absolute styles?

All we really need here is to change the direction of the arrows for desktop vs small tablet